pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1389,15 +1389,26 @@ func validateFirmware(field *k8sfield.Path, firmware *v1.Firmware) []metav1.Stat
 	return causes
 }
 
+func efiBootEnabled(firmware *v1.Firmware) bool {
```

Comment:
nit: We already have `util.IsEFIVMI`

Reply:
That's from [converter.go:2061](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/converter/converter.go#L2061)